xml/System.Net/HttpListener.xml

```diff
@@ -757,9 +757,7 @@
       <Parameters />
       <Docs>
         <summary>Waits for an incoming request as an asynchronous operation.</summary>
-        <returns>Returns <see cref="T:System.Threading.Tasks.Task`1" />.  
-  
- The task object representing the asynchronous operation. The <see cref="P:System.Threading.Tasks.Task`1.Result" /> property on the task object returns an <see cref="T:System.Net.HttpListenerContext" /> object that represents a client request.</returns>
+        <returns>The task object representing the asynchronous operation. The <see cref="P:System.Threading.Tasks.Task`1.Result" /> property on the task object returns an <see cref="T:System.Net.HttpListenerContext" /> object that represents a client request.</returns>
```

Comment:
At some point, the extended detail (on the Result property) for this and some of the following APIs should be moved to the Remarks section.